src/jenkins-pipeline-library.gdsl

```diff
@@ -149,6 +149,7 @@ contributor(notify) {
   method(name: 'mail', type: 'void', params: [config: Map], doc: 'Sends a mail notification.')
   method(name: 'mattermost', type: 'void', params: [config: Map], doc: 'Triggers a mattermost notification')
   method(name: 'mqtt', type: 'void', params: [config: Map], doc: 'Triggers a mqtt notification')
+  method(name: 'teams', type: 'void', params: [config: Map], doc: 'Triggers an MS Teams notification')
```

Comment:
Triggers a MS

Reply:
it's actually "an" here because MS starts with a "vowel sound", cf. https://www.quickanddirtytips.com/education/grammar/a-versus-an

> Similarly, you’d say, “She has an MBA, but chooses to work as a missionary,” because “MBA” starts with a vowel sound and “missionary” starts with a consonant sound.

Reply:
thanks!